src/transport/SessionManager.cpp

```diff
@@ -502,6 +463,24 @@ void SessionManager::SecureGroupMessageDispatch(const PacketHeader & packetHeade
 
     VerifyOrExit(!msg.IsNull(), ChipLogError(Inet, "Secure transport received NULL packet, discarding"));
 
+    // MCSP check
+    if (packetHeader.IsSecureSessionControlMsg())
+    {
+        if (packetHeader.GetDestinationNodeId().HasValue() && packetHeader.HasPrivacyFlag())
+        {
+            // TODO
```

Comment:
It should be nice to include more information in this TODO comment

Reply:
I thought that it was clear enough with the `//MCSP processing` a few line below.
I'll add the comment next time I fiddle with this section of code.